src/components/PreviewConcept/Concept.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { FC } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import styled from '@emotion/styled';
+import { spacing } from '@ndla/core';
+import { Remarkable } from 'remarkable';
+import {
+  NotionDialogContent,
+  NotionHeaderWithoutExitButton,
+  NotionDialogLicenses,
+  NotionDialogText,
+} from '@ndla/notion';
+import { Concept as ConceptType } from '../SlateEditor/editorTypes';
+import { getSrcSets } from '../../util/imageEditorUtil';
+
+const StyledBody = styled.div`
+  margin: 0 ${spacing.normal} ${spacing.small};
+  .c-tabs {
+    margin-left: 0;
+    margin-right: 0;
+  }
+  .react-tabs__tab-panel--selected {
+    animation-name: fadeInLeft;
+    animation-duration: 500ms;
+  }
+`;
+
+interface Props {
+  concept: ConceptType;
+}
+
+const Concept: FC<Props & tType> = ({ concept }) => {
+  const markdown = new Remarkable({ breaks: true });
```

Comment:
For å få samme oppførsel overalt.
```suggestion
  const markdown = new Remarkable({ breaks: true });
  markdown.inline.ruler.enable(['sub', 'sup']);
  markdown.block.ruler.disable(['list']);
```